CHANGELOG.md

```diff
@@ -11,6 +11,15 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 
+- feat: support for OpenMetrics registries, including exemplars
+
+- feat: out of the box support for exemplars populated with telemetry data on
+  default metrics when OpenTelemetry is loaded in the project
+
+- feat: new option for calling `observe()` and `inc()` methods on the histogram
+  and counter metric types that can be passed an object of format
+  `{labels: a, value: b, exemplarLabels: c}`
```

Comment:
should this be a separate PR?

Reply:
Right now the implementation is quite tightly coupled to the feature for exemplars. I though about making this more generic from the beginning but I was worried about the performance impact. I tried to provide 100% backwards compatibility and minimal performance impact with the change.

If I am to make a separate PR for it I would make this feature afailable for all metrics (not just counters and histograms) for the sake of consistency and I would 

Right now the call with `inc({value, labels, exemplarLabels})` can only be done if the metric was created with exemplars enabled.

If I make this call more generic - allow inc() and observe() to be called with an object - I would need to differentiate between the `inc({value, labels, exemplarLabels})` and the call that already exists `inc({labelNames})`. This might not be that bad, but it implies a check on the keys of the object every time inc/obs are used.

I am not sure which is the better aproach, let me know if you have feedback about this.

So the alternative to what we have now in the PR is: implement an object type check on all metric types and allow them to receive a single object containing all the relevant data for ins/obs operations. This might be useful on the long run, but it may as well be a feature that slows down the library for years until we need more fields like we do with exemplars (which are kind of a fringe feature anyway).

Reply:
>So the alternative to what we have now in the PR is: implement an object type check on all metric types and allow them to receive a single object containing all the relevant data for ins/obs operations. This might be useful on the long run, but it may as well be a feature that slows down the library for years until we need more fields like we do with exemplars (which are kind of a fringe feature anyway).

One thing we could do is drop support for passing `labels` (`inc({labelNames})` like you mention). That way you either pass a single value or an object. I don't `typeof arg === 'number'` adds much overhead, and then the object shape would be what we want.

Would require a major version of course, but that's not an issue. We wanna drop old nodes anyways